lib/portlayer/exec/container_cache.go

```diff
@@ -27,32 +34,55 @@ import (
 type containerCache struct {
 	m sync.RWMutex
 
-	// cache maps containerVM to container ID && infrastructure ID
+	// cache by container id
```

Comment:
If this is only containerID as a key then should we use `uid.UID` as the type instead of string?